bin/templates/project/build.gradle

```diff
@@ -19,16 +19,16 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 
 buildscript {
+    ext.kotlin_version = '1.3.50'
```

Comment:
is this the Android Studio default?

Reply:
Yes, if you create a blank "No Activity" project in Android Studio, it has in the project's `build.gradle`:

```
ext.kotlin_version = '1.3.50'
```